install/Knative-with-EKS.md

```diff
@@ -0,0 +1,105 @@
+# Knative Install on Elastic Container Service for Kubernetes (Amazon EKS)
```

Comment:
You'll want to add a link to this install guide from /install/README.md so that people can find it.

Reply:
Can you make that change in this PR, instead of creating a new PR? @gerrito333